pkg/app/piped/platformprovider/kubernetes/kubectl.go

```diff
@@ -28,8 +28,10 @@ import (
 )
 
 var (
-	errorReplaceNotFound = errors.New("specified resource is not found")
-	errorNotFoundLiteral = "Error from server (NotFound)"
+	errorReplaceNotFound    = errors.New("specified resource is not found")
+	errorNotFoundLiteral    = "Error from server (NotFound)"
+	errReplaceAlreadyExists = errors.New("resource already exists")
```

Comment:
```suggestion
	errResourceAlreadyExists = errors.New("resource already exists")
```
should be like this?